models/target.go

```diff
@@ -14,6 +14,7 @@ type Target struct {
 	NewRelicApiKey   string            `json:"new_relic_api_key"`
 	NewRelicAppId    string            `json:"new_relic_app_id"`
 	SlackUrl         string            `json:"slack_url"`
+	WebHooks         []Webhook         `json:"webhooks"`
```

Comment:
Super small nitpick: I think we want `Webhooks` here instead of `WebHooks` (note the `H`/`h`).